pyhsmm/models.py

```diff
@@ -478,9 +478,10 @@ def copy_sample(self):
 
     ### joblib parallel stuff here
 
-    def _joblib_resample_states(self,states_list,num_procs):
-        from joblib import Parallel, delayed
-        from . import parallel
+    def _joblib_resample_states(self,states_list,num_procs,backend):
+        from joblib import Parallel, delayed, parallel_backend
```

Comment:
do you need to register the dask backend by importing dask-ml? (`import dask_ml.joblib`) - [taken from this link](https://dask-ml.readthedocs.io/en/latest/joblib.html)

Reply:
Nope! Don't need dask-ml at all just a `dask.distributed` `client`.

https://ml.dask.org/joblib.html

Reply:
👍